django/contrib/admin/checks.py

```diff
@@ -957,6 +958,28 @@ def _check_actions_uniqueness(self, obj):
             )]
         return []
 
+    def _check_actions_base_classes(self, obj):
```

Comment:
This would lead to a warning for I’d guess 99% of all ModelAdmins out in the wild. I don’t think we can do that without a better reason.